test/tests/multiapps/steffensen/transient_main.i

```diff
@@ -0,0 +1,110 @@
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  nx = 5
+  ny = 5
+  parallel_type = replicated
+  uniform_refine = 1
+[]
+
+[Variables]
+  [u]
+  []
+[]
+
+[AuxVariables]
+  [v]
+  []
+[]
+
+[Kernels]
+  [diff]
+    type = CoefDiffusion
+    variable = u
+    coef = 0.1
+  []
+  [time]
+    type = TimeDerivative
+    variable = u
+  []
+  [force_u]
+    type = CoupledForce
+    variable = u
+    v = v
+  []
+[]
+
+[BCs]
+  [left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  []
+  [right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  []
+[]
+
+[Postprocessors]
+  [coupling_its]
+    type = NumFixedPointIterations
+    execute_on = 'initial timestep_end'
+  []
+  [unorm]
+    type = ElementL2Norm
+    variable = u
+  []
+[]
+
+[Executioner]
+  type = Transient
+  num_steps = 10
+  dt = 0.1
+  solve_type = PJFNK
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+  nl_abs_tol = 1e-14
+
+  fixed_point_algorithm = 'steffensen'
+  fixed_point_max_its = 30
+  transformed_variables = 'u'
```

Comment:
Should we autodetect this from the transfer block?

Reply:
u does not need to be transferred. Imagine a pp coupling with relaxation of the variables